docs/source/user/aerodyn/theory_ua.rst

```diff
@@ -465,13 +487,79 @@ Inputs
 See :numref:`ad_ua_inputs` for a description of the inputs necessary in the AeroDyn primary file (e.g. ``UAMod``) 
 
 See :numref:`airfoil_data_input_file` for a more comprehensive description of all the inputs present in the profile input file.
+Their default values are described in :numref:`UA_AFI_defaults`
 
 See :numref:`ua_notations` for a list of notations and definitions specific to unsteady aerodynamic inputs.
 
 An example of profile data (containing some of the unsteady aerodynamic parameters) is available here
 :download:`(here) <examples/ad_polar_example.dat>`.
 
 
+.. _UA_AFI_defaults:
+
+Calculating Default Airfoil Coefficients
+----------------------------------------
+
+The default value for ``cd0`` is the minimum value of the :math:`C_d` curve between :math:`\pm20` degrees angle of attack. 
+:math:`\alpha_{c_{d0}}` is defined to be the angle of attack where ``cd0`` occurs.
+
+After computing ``cd0``, the :math:`C_n` curve is computed by
+
+.. math::
+       C_{n}(\alpha) = C_l(\alpha) \cos\alpha + \left(C_d(\alpha) - c_{d0}\right) \sin\alpha
+
+The slope of the :math:`C_n` curve is computed as follows:
+   
+.. math::
+       C_{n}^{Slope}\left(\frac{\alpha_{i+1} + \alpha_i}{2}\right) = \frac{C_n(\alpha_{i+1}) - C_n(\alpha_i)}{\alpha_{i+1} - \alpha_i}
```

Comment:
Is this computed for a range of angle of attack and then few indices? How is this range determined? I'm guessing it's +/- 20 deg?

Reply:
The slope is computed for NumAlf-1 points (i.e., between all consecutive AoA specified in the polar). The smoothed values of this curve can have some weird behavior at the ends, but `alphaUpper` and `alphaLower` are far enough from the ends that it shouldn't matter.